### DISCUSSION TITLE
How to enable long form field with DB query or API data provider

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#7072] [Help] [answered]
Hi,
I was able to get long form field working with plain table changing the column property from type text to long form text.
However it didn't work when I create a query with the same column.

I was thinking perhaps I should change it in the schema of the query, but I am not able to find long form text in the drop down selection. I also check on schema from API response and didn't find the data type. 

Any ideas how I could work around this?

**Comment by melohagan** [comment, marked as answer]
Hi @micebrain 

There's no way to workaround this that I can think of. I've raised a feature request to add this here: https://github.com/Budibase/budibase/issues/7137

Why is it that you need the long form text type in this case?

**Reply by micebrain** [reply]
Thank you @melohagan for adding the feature request.
I am using a SQL query as a data source as I am displaying aggregated data as read only field but allowing the main table for editing. One of these fields is a long text field. But I am not able to use the `long form field` component.
It would be nice to have this feature. Currently I am trying to maximise the field by setting the longest possible width in the display.

**Reply by melohagan** [reply]
You could also try adding a little bit of custom CSS (https://docs.budibase.com/docs/custom-css).
Then you can set the height to a fixed height and even add a scrollbar if you like:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars

**Reply by micebrain** [reply]
Thank you for the tips! I will take a look at it.
